src/main/java/org/isf/patient/manager/PatientBrowserManager.java

```diff
@@ -289,124 +290,124 @@ public boolean isNamePresent(String name) throws OHServiceException {
 	}
 
 	/**
 	 * Method that merges patients and all clinic details under the same PAT_ID
-	 * 
+	 *
 	 * @param mergedPatient
 	 * @param patient2
 	 * @return true - if no OHServiceException occurred
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
-	public boolean mergePatient(Patient mergedPatient, Patient patient2) throws OHServiceException  {
-			if (mergedPatient.getBirthDate() != null &&
-					mergedPatient.getAgetype().compareTo("") == 0) {
-				//mergedPatient only Age
-				Date bdate2 = patient2.getBirthDate();
-				int age2 = patient2.getAge();
-				String ageType2 = patient2.getAgetype();
-				if (bdate2 != null) {
-					//patient2 has BirthDate
-					mergedPatient.setAge(age2);
-					mergedPatient.setBirthDate(bdate2);
-				}
-				if (bdate2 != null && ageType2.compareTo("") != 0) {
-					//patient2 has AgeType 
-					mergedPatient.setAge(age2);
-					mergedPatient.setAgetype(ageType2);
-				}
+	public boolean mergePatient(Patient mergedPatient, Patient patient2) throws OHServiceException {
+		if (mergedPatient.getBirthDate() != null && StringUtils.isEmpty(mergedPatient.getAgetype())) {
+			//mergedPatient only Age
+			Date bdate2 = patient2.getBirthDate();
+			int age2 = patient2.getAge();
+			String ageType2 = patient2.getAgetype();
+			if (bdate2 != null) {
+				//patient2 has BirthDate
+				mergedPatient.setAge(age2);
+				mergedPatient.setBirthDate(bdate2);
 			}
-			
-			if (mergedPatient.getAddress().compareTo("") == 0)
-				mergedPatient.setAddress(patient2.getAddress());
-			
-			if (mergedPatient.getCity().compareTo("") == 0)
-				mergedPatient.setCity(patient2.getCity());
-			
-			if (mergedPatient.getNextKin().compareTo("") == 0)
-				mergedPatient.setNextKin(patient2.getNextKin());
-			
-			if (mergedPatient.getTelephone().compareTo("") == 0)
-				mergedPatient.setTelephone(patient2.getTelephone());
-			
-			if (mergedPatient.getMother_name().compareTo("") == 0)
-				mergedPatient.setMother_name(patient2.getMother_name());
-			
-			if (mergedPatient.getMother() == 'U')
-				mergedPatient.setMother(patient2.getMother());
-			
-			if (mergedPatient.getFather_name().compareTo("") == 0)
-				mergedPatient.setFather_name(patient2.getFather_name());
-			
-			if (mergedPatient.getFather() == 'U')
-				mergedPatient.setFather(patient2.getFather());
-			
-			if (mergedPatient.getBloodType().compareTo("") == 0)
-				mergedPatient.setBloodType(patient2.getBloodType());
-			
-			if (mergedPatient.getHasInsurance() == 'U')
-				mergedPatient.setHasInsurance(patient2.getHasInsurance());
-			
-			if (mergedPatient.getParentTogether() == 'U')
-				mergedPatient.setParentTogether(patient2.getParentTogether());
-			
-			if (mergedPatient.getNote().compareTo("") == 0)
-				mergedPatient.setNote(patient2.getNote());
-			else {
-				String note = mergedPatient.getNote();
-				mergedPatient.setNote(patient2.getNote()+"\n\n"+note);
+			if (bdate2 != null && !ageType2.isEmpty()) {
```

Comment:
is there a reason why StringUtils is not used here?

Reply:
Nope; will add